doc/en/components/charts/chart-features-tooltips.md

```diff
@@ -7,8 +7,113 @@ mentionedTypes: []
 
 # $PlatformShort$ Chart Tooltips
 
-<!-- TODO combine
-category-chart-tooltip-types.md
-category-chart-tooltip-template.md
-data-chart-series-tooltips.md
--->
\ No newline at end of file
+This topic provides conceptual information about the tooltip types available on the $ProductName$ category chart component.
+
+<code-view style="height: 500px" 
+           data-demos-base-url="{environment:dvDemosBaseUrl}" 
+           iframe-src="{environment:dvDemosBaseUrl}/charts/category-chart-column-chart-with-tooltips" 
+           alt="$PlatformShort$ Tooltip Types Example" 
+           github-src="charts/category-chart/column-chart-with-tooltips">
+</code-view>
+
+<div class="divider--half"></div>
+
+## Tooltip Types
+
+The $ProductName$ category chart component can be configured to display Tooltips in the following ways:
+
+1. `Default` tooltips will display a tooltip for a single item when the pointer is positioned over it.
+1. `Item` tooltips will display a tooltip for each data item in the category that the pointer is positioned over.
+1. `Category` tooltips will display a grouped tooltip for all data points in the category that the pointer is positioned over.
+1. `None` will prevent tooltips from displaying.
+
+```razor
+<CategoryChart Height="500px" Width="700px"
+    DataSource="@DataSource"
+    ToolTipType="ToolTipType.Category"/>
+```
+
+```html
+<igx-category-chart
+    [dataSource]="data"
+    toolTipType="Category">
+</igx-category-chart>
+```
+
+```tsx
+ <IgrCategoryChart dataSource={this.state.data}
+    toolTipType="Category" />
+```
+
+```html
+<igc-category-chart
+    id="chart"
+    tool-tip-type="Category">
+</igc-category-chart>
+```
+
+<!--Angular-->
```

Comment:
why there is Angular build flag here?

Reply:
we don't have specifications for a tooltip template sample, nor is there ones for react, wc, or blazor.

Reply:
yes, we do
- [WC](https://github.com/IgniteUI/igniteui-wc-examples/blob/vnext/samples/charts/data-chart/series-tooltips/src/index.ts#L84)
- [REACT](https://github.com/IgniteUI/igniteui-react-examples/blob/vnext/samples/charts/data-chart/chart-overview/src/DataChartOverview.tsx#L118)
- [BLAZOR event trigger](https://github.com/IgniteUI/igniteui-blazor-examples/blob/vnext/samples/charts/data-chart/type-scatter-polygon-series/App.razor#L15)
- [BLAZOR event handler](https://github.com/IgniteUI/igniteui-blazor-examples/blob/vnext/samples/charts/data-chart/type-scatter-polygon-series/wwwroot/DataChartTypeScatterPolygonStyles.js#L34)

Reply:
Cannot use CategoryChart though. SeriesAdded fires but e.Series is not accessible and is always null. 

http://tfs.infragistics.local:8080/tfs/Engineering/NetAdvantage/_workitems/edit/272693

Since the Template section in the docs uses Category Chart for Angular we can merge this as-is or wait until the other platforms are fixed and samples are working properly.